Applications/MaterialManager/Contracts/Processing/Optimization/BookheightParameters.cs

```diff
@@ -20,6 +22,7 @@ public class BookHeightParameters : IContainsUnitSystemDependentProperties
         /// <summary>
         /// The deduction of the maximum saw blade projection.
         /// </summary>
+        [Obsolete]
```

Comment:
PLease add a message to the attribute what to use instead

Reply:
I added a message.